events/static/css/header.less

```diff
@@ -0,0 +1,5 @@
+
+.tabs {
+   width: 35% !important;
```

Comment:
To rewrite styles of components from react-material-ui library we decided to use !important instead of inline styles. Here is explanation part from documentation:
> Note that CSS properties defined inline are given priority over those defined in a CSS class. You need to use !important to take precedence over the inline style.